scripts/st2bootstrap-el8.sh

```diff
@@ -0,0 +1,760 @@
+
+  # Setup postgresql at a first time
+  sudo postgresql-setup initdb
+
+  # Configure service only listens on localhost
+  sudo sh -c "echo \"listen_addresses = '127.0.0.1'\" >> /var/lib/pgsql/data/postgresql.conf"
+
+  # Make localhost connections to use an MD5-encrypted password for authentication
+  sudo sed -i "s/\(host.*all.*all.*127.0.0.1\/32.*\)ident/\1md5/" /var/lib/pgsql/data/pg_hba.conf
+  sudo sed -i "s/\(host.*all.*all.*::1\/128.*\)ident/\1md5/" /var/lib/pgsql/data/pg_hba.conf
+
+  # Start PostgreSQL service
+  sudo systemctl start postgresql
+  sudo systemctl enable postgresql
+
+  cat << EHD | sudo -u postgres psql
+CREATE ROLE mistral WITH CREATEDB LOGIN ENCRYPTED PASSWORD '${ST2_POSTGRESQL_PASSWORD}';
+CREATE DATABASE mistral OWNER mistral;
+EHD
+}
+
+install_st2mistral() {
+  # 'st2' repo builds single 'st2' package and so we have to install 'st2mistral' from repo
+  if [ "$DEV_BUILD" = '' ] || [[ "$DEV_BUILD" =~ ^st2/.* ]]; then
+    sudo yum -y install ${ST2MISTRAL_PKG}
+  else
+    sudo yum -y install jq
+
+    PACKAGE_URL=$(get_package_url "${DEV_BUILD}" "el8" "st2mistral-.*.rpm")
+    sudo yum -y install ${PACKAGE_URL}
+  fi
+
+  # Configure database settings
+  sudo crudini --set /etc/mistral/mistral.conf database connection "postgresql+psycopg2://mistral:${ST2_POSTGRESQL_PASSWORD}@127.0.0.1/mistral"
+
+  # Setup Mistral DB tables, etc.
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+
+  # Register mistral actions.
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate | grep -v openstack | grep -v "ironicclient"
+
+  # start mistral
+  sudo systemctl start mistral
+}
+
+install_st2web() {
+  # Add key and repo for the latest stable nginx
+  sudo rpm --import http://nginx.org/keys/nginx_signing.key
+  sudo sh -c "cat <<EOT > /etc/yum.repos.d/nginx.repo
+[nginx]
+name=nginx repo
+baseurl=http://nginx.org/packages/rhel/7/x86_64/
+gpgcheck=1
+enabled=1
+EOT"
+
+  # Ensure that EPEL repo is not used for nginx
+  sudo sed -i 's/^\(enabled=1\)$/exclude=nginx\n\1/g' /etc/yum.repos.d/epel.repo
+
+  # Install nginx
+  sudo yum install -y nginx
+
+  # Install st2web
+  sudo yum install -y ${ST2WEB_PKG}
+
+  # Generate self-signed certificate or place your existing certificate under /etc/ssl/st2
+  sudo mkdir -p /etc/ssl/st2
+
+  sudo openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt \
+  -days 365 -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information Technology/CN=$(hostname)"
+
+  # Remove default site, if present
+  sudo rm -f /etc/nginx/conf.d/default.conf
+
+  # Copy and enable StackStorm's supplied config file
+  sudo cp /usr/share/doc/st2/conf/nginx/st2.conf /etc/nginx/conf.d/
+
+  sudo systemctl restart nginx
+  sudo systemctl enable nginx
+}
+
+install_st2chatops() {
+  # Temporary hack until proper upstream fix https://bugs.centos.org/view.php?id=13669
```

Comment:
I believe this hack is EL7-related only. Let's please remove it and we'll see if we need it.